artifacts/examples/shoryuken-worker-wpa-multiqueue.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: k8s.practo.dev/v1
+kind: WorkerPodAutoScalerMultiQueue
+metadata:
+  name: shoryuken-worker-wpa-multiqueue-development
+spec:
+  minReplicas: 1
+  maxReplicas: 10
+  deploymentName: shoryuken-worker-development
+  queues:
```

Comment:
Here is an example of the new spec for wpa-multiqueue. I have introduced a new variable called `enqueueRate` per job. This is then used internally to calculate `targetMessagesPerWorker`.